### PR TITLE
closes #110

### LINKED ISSUE #110: Add read(index, show) functionality for weather

### PR DESCRIPTION
**Pull request by MikelSage**
- adds show and index pages for weather_conditions

### CODE REVIEW

**Review by DesTodo — APPROVED** [review]
Looks great. Let's merge this commit after we merge the updated seed.rb. I seeded the database with zip code and I believe this migration should be fine, but I think we should merge seed.rb first. @MikelSage @erose357

**Comment by MikelSage** [comment]
@DesTodo I'm thinking that we should probably merge this first, then edit the seed to remove the zipcode column as well since we never actually use it. All the data analysis for weather is by date.

**Pull request merged**
